aee.js

```diff
@@ -141,3 +141,15 @@ Array.prototype.min = function (comparer) {
   }
   return result
 }
+
+Array.prototype.flatten = function () {
+  let copy = this.slice()
+  let len = copy.length
+  for (let i = 0; i < len; i += 1) {
+    if (Array.isArray(copy[i])) {
+      len += copy[i].length - 1
+      copy.splice(i, 1, ...copy[i])
+    }
+  }
+  return copy
```

Comment:
If you take a look at the whole file this `copy` guy is the black in the rice, I think all the `result` boys will bully him.